irohad/execution/impl/query_execution.cpp

```diff
@@ -332,6 +320,17 @@ QueryProcessingFactory::executeGetSignatories(
   auto response = QueryResponseBuilder().signatoriesResponse(*signs);
   return response;
 }
+
+QueryProcessingFactory::QueryResponseBuilderDone
+QueryProcessingFactory::executeGetPendingTransactions(
+    const shared_model::interface::GetPendingTransactions &query,
+    const shared_model::interface::types::AccountIdType &query_creator) {
+  std::vector<shared_model::proto::Transaction> txs;
```

Comment:
Always empty?

Reply:
I think it is temporary unless query execution is implemented. Please leave a comment about that

Reply:
Sorry, haven't noticed that. Maybe you consider implementing mst? It's already implemented

Reply:
It is ok. Now we have common understanding. TODO about missing implementation is placed on a line below.